.github/pull_request_template.md

```diff
@@ -1,39 +1,42 @@
+## Type Of Change
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Any modification in the test cases
+- [ ] Any modification in the documentation
+
 ### Checklist:
 
 * [ ] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
-* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change
-* [ ] I have tested these changes and added tests that prove my fix is effective or that my feature works
```

Comment:
This should be kept

Reply:
The first one is kept, the second one has been rephrased.